app/services/idv/actions/verify_document_status_action.rb

```diff
@@ -70,14 +69,13 @@ def process_result(async_state)
 
       def verify_document_capture_session
         return @verify_document_capture_session if defined?(@verify_document_capture_session)
-        @verify_document_capture_session =
-          if hybrid_flow_mobile?
-            document_capture_session
-          else
-            DocumentCaptureSession.find_by(
-              uuid: flow_session[verify_document_capture_session_uuid_key],
-            )
```

Comment:
TIL that we had previously been creating a distinct `DocumentCaptureSession` record for the document capture async verification:

https://github.com/18F/identity-idp/blob/ec17ed1f7ec2ae0536268e716823d86a2ae3515e/app/services/idv/actions/verify_document_action.rb#L38

This is not the same record as the one we populate into the document capture page and attach to all its API requests, which is the one created at the first welcome step:

https://github.com/18F/identity-idp/blob/ec17ed1f7ec2ae0536268e716823d86a2ae3515e/app/services/idv/steps/welcome_step.rb#L8

https://github.com/18F/identity-idp/blob/ec17ed1f7ec2ae0536268e716823d86a2ae3515e/app/views/idv/shared/_document_capture.html.erb#L21

https://github.com/18F/identity-idp/blob/ec17ed1f7ec2ae0536268e716823d86a2ae3515e/app/javascript/packs/document-capture.jsx#L105-L108

I'm assuming this was intentional, though I'm not really familiar with the reasoning. And if we do want to continue doing creating distinct records, I've a few thoughts:

1. Was it not an issue / inconsistency that the hybrid mobile would be [using the welcome step session](https://github.com/18F/identity-idp/blob/ec17ed1f7ec2ae0536268e716823d86a2ae3515e/app/services/idv/actions/verify_document_action.rb#L34-L36)?
2. Maybe we should create the distinct record when the document capture step is visited, so that we can [initialize the view](https://github.com/18F/identity-idp/blob/ec17ed1f7ec2ae0536268e716823d86a2ae3515e/app/views/idv/shared/_document_capture.html.erb#L21) with that one instead of the one created at the welcome step?

cc @zachmargolis @stevegsa for thoughts / insights (via git blame'd #4313, #4665)

Reply:
@zachmargolis Thanks. Curious, though: In that framing, what's the purpose of [the session created during the welcome step](https://github.com/18F/identity-idp/blob/346e59fe7959a4b08fbf6cef7969900770f3b563/app/services/idv/steps/welcome_step.rb#L8)?

Reply:
I'm reaching the limits of my memory and my ability to unpack what this was supposed to do....I think the one created at the welcome step was probably meant to be shared across desktop + mobile in the hybrid flow